test/key_test.py

```diff
@@ -24,19 +24,24 @@ def test_import(self):
         "does it import"
         import pygame.key
 
-    def todo_test_get_focused(self):
-
-        # __doc__ (as of 2008-08-02) for pygame.key.get_focused:
-
-        # pygame.key.get_focused(): return bool
-        # true if the display is receiving keyboard input from the system
-        #
-        # This is true when the display window has keyboard focus from the
-        # system. If the display needs to ensure it does not lose keyboard
-        # focus, it can use pygame.event.set_grab() to grab all input.
-        #
-
-        self.fail()
+    def test_get_focused(self):
+        focused = pygame.key.get_focused()
+        if not SDL1:
+            self.assertFalse(focused) #No window to focus
+        self.assertIsInstance(focused, int)
+        display_sizes = pygame.display.list_modes()
+        if display_sizes == -1:
+            display_sizes = [(500, 500)]
+        if not display_sizes:
+            return
```

Comment:
Does returning here cause this test to fail? Genuinely not sure.

Reply:
No, returning causes the test to succeed. The idea I had was that if I was unable to get a fullscreen window to test, I would just stop the test.